db/migrate/20250117112840_create_professional_body_membership.rb

```diff
@@ -0,0 +1,15 @@
+class CreateProfessionalBodyMembership < ActiveRecord::Migration[7.2]
+  # rubocop:disable Rails/ThreeStateBooleanColumn
+  def change
+    create_table :professional_body_memberships, id: :uuid do |t|
+      t.timestamps
+      t.string :name
+      t.string :membership_type
+      t.string :membership_number
+      t.string :date_membership_obtained
```

Comment:
This appears to be a number on the form (the year) so maybe this should be  a number and stored as 'year membership obtained'?